samples/basic/blink_led/nrf52832_mdk.overlay

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Why do you need this file? For boards with nRF52 the HW PWM driver is enabled by default.

Reply:
SW-PWM setting "CONFIG_PWM_NRF5_SW_0_CLOCK_PRESCALER" was set and used for this example on the nrf52832_mdk board earlier. I did not want to change that, so I updated and added this file. Now that it has changed, no need to keep this overlay since HW PWM is used. 
File removed.